aspnetcore/security/authentication/certauth.md

```diff
@@ -551,6 +553,18 @@ namespace AspNetCoreCertificateAuthApi
 
 <a name="occ"></a>
 
+## Certificate validation cachine
```

Comment:
There is probably conditional display instruction to only display this content when viewing 5.0 docs, e.g.

`::: moniker range=">= aspnetcore-3.0"`

`::: moniker-end`

Should use this so we don't need to mention the version it is available from. I don't know the variable is called in 5.0

Reply:
As James mentioned, you can toggle the visibility of content for 5.0 and later with the following:

```
::: moniker range=">= aspnetcore-5.0"

5.0+ content here

::: moniker-end
```